examples/ansible.cfg

```
@@ -77,14 +77,14 @@
 #stdout_callback = debug
 
 
-# Ansible ships with some plugins that require whitelisting,
+# Ansible ships with some plugins that require enabling
 # this is done to avoid running all of a type by default.
 # These setting lists those that you want enabled for your system.
 # Custom plugins should not need this unless plugin author disables them
 # by default.
 #
 # Enable callback plugins, they can output to stdout but cannot be 'stdout' type.
-#callback_whitelist = timer, mail
+#callback_enabled = timer, mail
```

Can this be made plural?
```suggestion
#callbacks_enabled = timer, mail
```

---

we can add new one and deprecate existing